tox.ini

```diff
@@ -2,9 +2,10 @@
 envlist = pep8,py27,py3
 
 [testenv]
-deps = mock
-commands =
-    python -m unittest discover --top-level-directory=. --start-directory=tests/
+deps =
+    mock
+    pytest
+commands = pytest tests/
```

Comment:
Basically `pytest` makes it possible to see exact result of failing comparison. That way we don't need to debug print around in Travis :wink: